### ISSUE TITLE
Elites lvls 70 - 72 Wrath Masters and Fel Soilders spawning at 2nd "portal" of weaken the Ramparts

### ISSUE THREAD

**Issue by AGuitar-God** [chromiecraft/chromiecraft#7643]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

61-64

### Current Behaviour

While doing the quest "Disrupt their Reinforcements" on alliance, I was killing mobs around the northern two portals per quest requirements  to get the necessary stone quest items to destroy these portals.  Suddenly a group of 5 elites (4 level 70 Fel Soldier elites and one level 72 elite Wrath Master teleported in right next to me.  I was multiboxing 5 Hardcore toons at the time and they decimated 4 out of 5 characters (the pally bubble hearthed away) in seconds.  This is the exact portal that had the problem with the pit commander sitting on top of it for months.  According to Wowhead, Wrath masters and fel soldiers are only supposed to be attacking the Stairs of Destiny and should not be in the area for "Disrupt the Reinforcements", this to me seems clearly to be a bug.  Please note that I am aware there is a level 60 elite that roams around and can teleport in using these portals, I could have easily handled him as it is only one mob (and I was multiboxing 5), but this is not what teleported in.   **I have a video clearly showing the incident, but the video is too big to attach here, however it is attached in my request ticket to resurrect my hardcore toons and can be viewed under this ticket: #account-ticket-2796-691311603294929016.**  Please note I am told that my characters cannot be resurrected until this issue is confirmed a bug.  Thank you,

### Expected Blizzlike Behaviour

What should have happened is I kill some mobs, pick up the stones they drop and click on the portal to destoy them, its not that hard of a quest.  

### Source

Wowhead is my source.

### Steps to reproduce the problem

Do what I did in the video?  Kill mobs by the portal I guess.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/5f020de5466e3894914fe424229c4b9f3d138b2f

### Operating system

win 10

### Modules

- [mod-1v1-arena](https://github.com/azerothcore/mod-1v1-arena)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-anticheat](https://github.com/azerothcore/mod-anticheat)
- [mod-arena-3v3-solo-queue](https://github.com/azerothcore/mod-arena-3v3-solo-queue)
- [mod-arena-replay](https://github.com/azerothcore/mod-arena-replay)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-breaking-news-override](https://github.com/azerothcore/mod-breaking-news-override)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-costumes](https://github.com/azerothcore/mod-costumes)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-dmf-switch](https://github.com/azerothcore/mod-dmf-switch)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-npc-spectator](https://github.com/gozzim/mod-npc-spectator)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-eventScripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-ExchangeNpc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-RecruitAFriend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-SendAndBind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-TempAnnouncements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-Zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by TheSCREWEDSoftware** [comment]
@AGuitar-God are you able to share the same video of the one you shared in the ticket here?

**Comment by AGuitar-God** [comment]
Yes I still have it, it is 44.5 megs so too big to attach to an email.
Let me know if you wish it sent to you via another method.Thanks

    On Saturday, January 11, 2025 at 12:56:24 PM PST, FlyingArowana ***@***.***> wrote:  
 
 


@AGuitar-God are you able to share the same video of the one you shared in the ticket here?

**Comment by TheSCREWEDSoftware** [comment]
> Yes I still have it, it is 44.5 megs so too big to attach to an email. Let me know if you wish it sent to you via another method.Thanks On Saturday, January 11, 2025 at 12:56:24 PM PST, FlyingArowana ***@***.***> wrote: @AGuitar-God are you able to share the same video of the one you shared in the ticket here? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>

Where you have a chance to drop the video either on youtube and leave the link here or try to upload it to github directly.
